src/raygui.h

```diff
@@ -257,7 +257,7 @@
         float y;
     } Vector2;
 
-    // Vector3 type                 // -- ConvertHSVtoRGB(), ConvertRGBtoHSV()
```

Comment:
I would prefer to avoid removing this piece of information, it's important for me to know why this struct is requried